marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -19,6 +19,7 @@ describe('json scenario for snippet on browse documents page', () => {
     cy.contains(Application.title);
     cy.loginAsDeveloper().withRequest();
     cy.waitUntil(() => toolbar.getExploreToolbarIcon()).click();
+    browsePage.getSelectedEntity().should('contain', 'All Entities');
```

Comment:
Is there a better way to test this, probably RTL unit test than asserting in a before Each block.
An e2e test may not worry about default option being selected always and if beforeEach fails Cypress doesnt run any tests.

Reply:
You can probably enhance existing test to verify default value:

expect(getByTestId('entity-select')).toBeInTheDocument();

Reply:
will verify on unit test level.